nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-core/src/main/java/org/nuxeo/connect/client/status/ConnectStatusHolder.java

```diff
@@ -54,6 +54,15 @@ protected ConnectRegistrationService getService() {
         return Framework.getLocalService(ConnectRegistrationService.class);
     }
 
+    /**
+     * @deprecated Deprecated since 9.2, use {@link #isRegistered()} instead.
+     */
+    @Deprecated
+    public boolean isRegistred() {
+        // no cache needed
+        return getService().isInstanceRegistered();
+    }
+
     public boolean isRegistered() {
```

Comment:
It's weird, I cannot see the commit that you've pointed. It should have been amended. The `@since` has already been added in the fixup commit.